groovy-language-server/src/main/java/com/palantir/groovylanguageserver/CompilerWrapper.java

```diff
@@ -37,17 +39,31 @@
 
     /**
      * Returns a mapping from absolute path of source file to symbols located within these source files.
-     * @return the list of symbols
+     * @return the set of symbols
      */
     Map<String, Set<SymbolInformation>> getFileSymbols();
 
+
+    /**
+     * Returns a mapping from the class or interface name to symbols which reference those types.
+     * @return the set of references
+     */
+    Map<String, Set<SymbolInformation>> getReferences();
+
+    /**
+     * Returns the locations of the symbols that reference the symbol defined by the given params.
+     * @param params the parameters used to filter down which symbol is referenced
+     * @return the set of references
+     */
+    Set<Location> findReferences(TextDocumentPositionParams params);
```

Comment:
I made it return `Location` just because that's what `GroovyTextDocumentService#references` needs to return to the client, but I agree that maybe this interface should return `Set<SymbolInformation>` to be more consistent with the other call.